fw/http.c

```diff
@@ -1541,6 +1543,13 @@ tfw_http_req_fwd_send(TfwSrvConn *srv_conn, TfwServer *srv, TfwHttpReq *req,
 	req->jtxtstamp = jiffies;
 	tfw_http_req_init_ss_flags(srv_conn, req);
 
+	/* We set TFW_CONN_B_UNSCHED on server connection. New requests must
+	 * not be scheduled to it. It will be used only for websocket transport.
+	 * If upgrade will fail, we clear it. */
```

Comment:
Looks good for met. Could you please just add a comment why we don't have two racing `tfw_http_req_fwd_send()` here? (In my understanding because the function is called under `fwd_qlock`).